README.md

```diff
@@ -75,6 +75,7 @@ If you cannot provide a valid website for your company your PR will not be accep
 - [SPI Nepal](http://www.softvision.com/) | Thapathali, Kathmandu | Provider of Digital Solutions, Mobility, Analytics, Merchandise Planning, In-Store Engagements and Quality Assurance
 - [SUBISU CABLENET (P.) LTD.](http://subisu.net.np/) | Jhamshikhel, Lalitpur | Internet, Network and Television Service Provider
 - [Three Monks](http://www.threemonk.com/) | Sinamangal, Kathmandu | Carter the growing needs in the space of technical expertise with special focus on Digital Financial Services, Money Transfer Business and Entertainment Companies
+- [Tech101](https://www.tech101.co.uk) | Kamaladi ganeshthan, Kathmandu | Web design, development and digital marketing
```

Comment:
Hello @dineshkhadka - Let's do `Kamaladi, Kathmandu` to make it consistent.

Also, `Tech101` should be above `Three Monks` alphabetically.